sklearn/decomposition/factor_analysis.py

```diff
@@ -347,3 +359,36 @@ def score(self, X, y=None):
             Average log-likelihood of the samples under the current model
         """
         return np.mean(self.score_samples(X))
+
+    def _rotate(self, method="varimax", n_components=None, tol=1e-6):
+        "Rotate the factor analysis solution."
+        implemented = {"varimax", "quartimax"}
+        if method in implemented:
+            self.components_ = _ortho_rotation(
+                self.components_.T, method=method, eps=tol)[:n_components]
```

Comment:
Please update the documentation of tol to note its use here.

Reply:
tol is actually not exposed, this is purely internal ... The tol for the exposed function is for the svd. Or did I misunderstand something?